data/geany.glade

```diff
@@ -3498,6 +3516,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="tooltip_text" translatable="yes">Display height in rows for the autocompletion list</property>
+                                        <property name="invisible_char">●</property>
```

Comment:
I don't think I had such issues last time I modified that file, so IMO it requires at least an investigation.  Not to add and remove that every now and then.

Sure it's generated, but it requires reviewing as it's the source file. Less noise means less burden on the review.